lib/server.js

```diff
@@ -728,6 +728,7 @@ WalletService.prototype._selectTxInputs = function(txp, cb) {
           if (!bitcoreError) {
             txp.inputPaths = _.pluck(txp.inputs, 'path');
             txp.fee = bitcoreTx.getFee();
+            $.checkState(txp.fee < 1e8, 'Fees are too high!');
```

Comment:
We generally trust Bitcore. This check is in place as a last resort to protect agains some crazy unlikely bug. If we use Bitcore to compute tx size we would be relaying on the very same code we need to protect ourselves from. Does this make sense?
Anyways, I think a better place for this check would be here: https://github.com/bitpay/bitcore-wallet-utils/blob/master/lib/walletutils.js#L177 as this code is used by both server & client. What do you think?